src/main/java/com/google/api/generator/engine/writer/ImportWriterVisitor.java

```diff
@@ -362,11 +362,13 @@ public void visit(TryCatchStatement tryCatchStatement) {
     statements(tryCatchStatement.tryBody());
 
     Preconditions.checkState(
-        !tryCatchStatement.isSampleCode() && tryCatchStatement.catchVariableExpr() != null,
+        !tryCatchStatement.isSampleCode() && !tryCatchStatement.catchVariableExprs().isEmpty(),
         "Import generation should not be invoked on sample code, but was found when visiting a"
             + " try-catch block");
-    tryCatchStatement.catchVariableExpr().accept(this);
-    statements(tryCatchStatement.catchBody());
+    for (int i = 0; i < tryCatchStatement.catchVariableExprs().size(); i++) {
```

Comment:
super minor: I guess these technically should be done via two simultaneous iterators, but good enough.